vmdb/app/models/host_openstack_infra.rb

```diff
@@ -1,4 +1,6 @@
 class HostOpenstackInfra < Host
+  has_many                  :openstack_system_services, :dependent => :destroy
```

Comment:
I think is shoudl be 

has_many :openstack_host_services, :dependent => :destroy

since its openstack_host_services model

and you can drop the spaces